R/mod_utilization.R

```diff
@@ -0,0 +1,151 @@
+#' utilization UI Function
+#'
+#' @description A shiny Module.
+#'
+#' @param id,input,output,session Internal parameters for {shiny}.
+#'
+#' @noRd 
+#'
+#' @importFrom shiny NS tagList 
+# mod_UI
+mod_utilization_ui <- function(id){
+  ns <- shiny::NS(id)
+  shiny::fluidPage(
+    ui_header_row(),
+    ui_row(box(
```

Comment:
See `box` argument for `ui_row` instead of this `box`, since this could arbitrarily be any package with `box`. Also, can we use namespaces when declaring a function that isn't directly imported by shiny so it's clear what package is being called?